### ISSUE TITLE
Possible conflict with Video Combine node for the preview?

### ISSUE THREAD

**Issue by Garin2-colab** [SamKhoze/ComfyUI-DeepFuze#32]
When I install Deepfuze the preview of 'Video combine' node doesn't work. it still saves the file but no preview. When I diable Deepfuze it works again.

**Comment by SamKhoze** [comment]
Hello @Garin2-colab,
I think It may be issue regrading the Video combine node We will definitely  try to look into it

**Comment by pmarmotte2** [comment]
I don't think it is an issue with VHS nodes,  installing Deepfuze seems to break any "preview" would it be text or video.
